src/toil/provisioners/gceProvisioner.py

```diff
@@ -535,11 +535,16 @@ def addNodes(self, nodeType, numNodes, preemptable):
     def getProvisionedWorkers(self, nodeType, preemptable):
         entireCluster = self._getNodesInCluster(nodeType=nodeType)
         logger.debug('All nodes in cluster: %s', entireCluster)
-        workerInstances = [i for i in entireCluster if i.private_ips[0] != self.leaderIP]
+        workerInstances = []
+        for instance in entireCluster:
+            preemtible = False
```

Comment:
preemptable and preemtible!!! One of these needs to be changed

Reply:
I have followed the spelling conventions used by Toil, which have consistently mis-spelled it.

Reply:
`preemtible` isn't in the code: https://github.com/BD2KGenomics/toil/search?utf8=%E2%9C%93&q=preemtible&type= .  I agree this should be changed rather than creating even more variants of the misspelling.